runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -90,4 +90,24 @@ default boolean syncPanel()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = "npcKillChatMessage",
+		name = "Show chat message for NPC kills",
+		description = "Adds a chat message with monster name and kill value when receiving loot from an NPC kill."
+	)
+	default boolean npcKillChatMessage()
```

Comment:
I would make this config enum (OFF, NPC, PLAYER, BOTH)

Reply:
Do we expect chat messages would be desired for other events besides PVP/NPC kills? If so, a dropdown would not scale for that.

Reply:
Oh right, now we also support bunch of other stuff